pkg/utils/utils.go

```diff
@@ -51,6 +51,16 @@ func EnableArpAndNdiscNotify(ifName string) error {
 	return nil
 }
 
+// EnableOptimisticDad enables IPv6 /proc/sys/net/ipv6/conf/$ifName/optimistic_dad
+func EnableOptimisticDad(ifName string) error {
```

Comment:
lets not return error if path does not exist.

Reply:
could you elaborate why? If the path does not exist (which is most likely because the interface was just renamed or ipv6 module is not loaded), it's still an error. Granted, the caller ignores the return value anyway.

Reply:
my though was: if path does not exist there is nothing to enable - so this func should be a no-op.
since ipv6 modules may not me loaded. instead of always trying to write.

lets keep as is.

Reply:
Since the caller ignores the error it's kinda moot anyway.

It the caller ever will do something about the error, we know better what they expect. Then we can fine tune the desired error behavior. Which will not be a problem, since this is only internal API that can change.